core/deployment/src/main/java/io/quarkus/deployment/steps/ConfigurationSetup.java

```diff
@@ -164,57 +155,31 @@
             NameIterator.class);
     private static final MethodDescriptor ARDCS_CTOR = MethodDescriptor.ofConstructor(AbstractRawDefaultConfigSource.class);
 
+    private static final MethodDescriptor CS_POPULATE_CONVERTERS = MethodDescriptor.ofMethod(ConverterSupport.class,
+            "populateConverters", void.class, ConfigBuilder.class);
+
     private static final String CONFIG_ROOTS_LIST = "META-INF/quarkus-config-roots.list";
     private static final String[] NO_STRINGS = new String[0];
 
     public ConfigurationSetup() {
     }
 
-    @BuildStep
-    public void setUpConverters(BuildProducer<ConfigurationCustomConverterBuildItem> configurationTypes) {
```

Comment:
@dmlloyd Ok. I will change `DEFAULT_CONVERTER_PRIORITY` in `ConverterSupport`  to be 100 instead of 200 and add `@Priority` to all Quarkus explicit converters.

Reply:
@dmlloyd This is now done. I added `@Priority(DEFAULT_QUARKUS_CONVERTER_PRIORITY)`, on all Quarkus converters, which has a value of 200. All converters made by users will have a default priority of 100.